docs/integrations/zapier.mdx

```diff
@@ -0,0 +1,107 @@
+
+Svix can auto-generate a Zapier integration package for your environment that follows this pattern.
+It also comes with webhook verification baked in and can be extended for custom authentication schemes with your service or additional actions/triggers.
+
+:::note Information
+Zapier integrations can be defined on the Zapier UI or as a JavaScript project (defined through the Zapier CLI platform). The auto-generated integration package is a Node.js package. We recommend reading through the [How to Use REST Hooks in Zapier CLI](https://platform.zapier.com/cli_tutorials/resthooks) tutorial provided by Zapier to get a basic understanding of the structure.
+:::
+
+## Prerequisites
+
+### Setup event type schemas
+
+Before setting up a Zapier integration, we strongly recommend configuring event types with schemas and examples as those will be embedded in the auto-generated integration package.
+
+You can follow the [Your first event type schema](/tutorials/event-type-schema) tutorial and refer to the [Event Types](/tutorials/event-type-schema) docs to set this up.
+
+### Download dependencies
+
+The integration requires Node.js v14 as that is the version used by the Zapier platform at runtime. Install the Zapier CLI globally:
+
+```
+npm install -g zapier-platform-cli
+```
+
+Make a Zapier platform account and login:
+```
+# Login with username and password
+zapier login
+
+# Login with SSO
+zapier login --sso
+```
+
+## Create or upgrade an integration
+
+:::note Upgrades & Modifications
+If you're upgrading the integration after making customizations or adding new event types, you may need to reconcile differences between the last version deployed and the latest version downloaded. We recommend using a version control system like Git to manage the history.
+:::
+
+### Download the auto-generated integration package
+
+You can download an auto-generated integration package from the [integrations page](https://dashboard.svix.com/integrations) on the dashboard shown below.
+
+The download form requires a project name which corresponds to the NPM project name, and a version number which corresponds to both the NPM projection version and the Zapier integration version.
+
+![integration-zapier](../img/integration-zapier.png)
+
+### Build and deploy
+
+Once you've downloaded the auto-generated integration package, download the package dependencies:
+
+```
+npm install
+```
+
+Create a new Zapier integration using the CLI or link an existing one to the package. If you are upgrading, this will overwrite an unreleased integration or require a version bump for a released one.
+
+```
+# Create a new Zapier integration
+zapier register
+
+# Link an existing Zapier integration to upgrade it
+zapier link
+```
+
+Build & deploy the integration:
+```
+zapier push
+```
+
+Finally, try it out! From the account where you deployed the integration, you can create a Zap and test the integration. It won't be generally available on Zapier until it's released.
```

Comment:
💡  One idea would be to include either a screenshot of a basic Zap here or we could eventually add a separate tutorial on Svix zaps and link to that.

Reply:
Good call! Details on the zap UX is missing.

Also need to add some details on the auth flow for a zap.